ozw-network-visualization-card.js

```diff
@@ -109,7 +109,8 @@ class OZWNetworkVisualizationCard extends HTMLElement {
 
   _getLQI(lqi) {
     if (lqi > 192) {
-      return { color: "#17ab00", highlight: "#17ab00" };
+      //darken unselected edges, and brightly mark edges of the selected node (or edge)
+      return { color: "darkred", highlight: "yellow" };
```

Comment:
I would prefer if we leave the previous green color instead of the red one for two reasons: 
- it works better on both the default light and dark themes of HA
- if/when we have link quality if, we already have the green color set here
Also, the green contrasts with the yellow well enough.

Reply:
Sure, but how do I change this? reedit my code and try a PR again? sorry, this is new to me.

Reply:
You just create a new commit and push to the same branch, but I guess you've figured that out already.